curp/tests/it/server.rs

```diff
@@ -299,7 +299,7 @@ async fn shutdown_rpc_should_shutdown_the_cluster() {
     assert!(matches!(res, Err(ClientError::ShuttingDown)));
 
     let collection = collection_task.await.unwrap();
-    sleep_secs(3).await; // wait for the cluster to shutdown
+    sleep_secs(7).await; // wait for the cluster to shutdown
```

Comment:
Why should we enlarge the sleep time in this test case. The code covered by this test case doesn't involve the parts you modified in this pr.

Reply:
done

Reply:
After replacing it to the new client, I will test this time again.

Reply:
Are you planning to complete this test in this PR, or in another PR? If it's in this PR, please resolve this conversation after the test is completed. If it's in another PR, please open an issue to track this matter.

Reply:
I've submitted a new issue: https://github.com/xline-kv/Xline/issues/596